xseed.sh

```diff
@@ -8,7 +8,7 @@
 
 # Load environment variables from .env file if it exists
 # in the same directory as this bash script
-VERSION='3.1.0'
+VERSION='3.2.0'
```

Comment:
```suggestion
VERSION=4.0.0'
```

This is a non-backwards compatible breaking change with the variables and as such requires a major version bump.